### DISCUSSION TITLE
Importing Recipes from other apps

### DISCUSSION THREAD

**Discussion by DrSpaldo** [TomBursch/kitchenowl#155] [Q&A]
Hi, I am not sure if I have totally missed it but after having a dig around in the KitchenOwl site and docs, I can't see the method of importing recipes from another app. I use TheCookBook, Mealie & Chowdown. Is there a way I can transfer my recipes to KitchenOwl?

**Comment by TomBursch** [comment]
I've worked a lot on import/exporting stuff the last week. But currently only a very specific JSON format is supported, and I'm not sure how hard it is to get the export of other apps in that format. I wouldn't consider the import API stable, so in the near future the format might change as well. 

So to answer your question, right now sadly there is no easy way to import recipes from other apps.

**Comment by seanmcilvenna** [comment]
I'd use it today if I could import recipes. I even took a stab at trying to derive the JSON format. Eventually just fell flat with an  exception I had no clue what to do with `TypeError: The view function for 'api.household.import.importData' did not return a valid response. The function either returned None or ended without a return statement.`. I don't have the patience to hand enter all our recipes, unfortunately. Otherwise it seems like a great tool.
